### PR TITLE
Replace sanity url builder with images from the filesystem

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard**
## Summary

Parts of the codebase are talking to Sanity through the URL builder.
This PR replaces some of them with our own image component that builds a srcSet and serves images from the local file system.

## Motivation

Avoiding the Sanity CDN limits.

## Unresolved questions
- I'm quite sure the types could be better. Because I'm using a wrapper around  it seems TS does not understand properties such as srcSet, loading and alt natively anymore. I don't mind postponing these enhancements until after an initial release.

### CODE REVIEW

File: packages/app/src/components-styled/article-detail.tsx [hunk 1]
```diff
@@ -40,8 +40,6 @@ export function ArticleDetail({ article }: ArticleDetailProps) {
           <PortableText blocks={article.intro} />
         </Box>
 
-        {/* <Image node={article.cover} /> */}
```
Comment (VWSCoronaDashboard1) [inline packages/app/src/components-styled/article-detail.tsx:43]: remove comment

**Review by VWSCoronaDashboard1 — CHANGES_REQUESTED** [review]
This PR isnt complete yet IMHO. There are still the ArticleTeaser and EditorialTeaser components that make use of the 'old' image functionality.
Wouldn't this PR break these?

File: packages/app/src/components-styled/article-detail.tsx (hunk 1 above)
Comment (VWSCoronaDashboard) [reply packages/app/src/components-styled/article-detail.tsx:43]: I don't see this in my codebase, weirdly enough.

Comment (VWSCoronaDashboard1) [reply packages/app/src/components-styled/article-detail.tsx:43]: GREMLINS!

**Comment by VWSCoronaDashboard** [comment]
> This PR isnt complete yet IMHO. There are still the ArticleTeaser and EditorialTeaser components that make use of the 'old' image functionality.
> Wouldn't this PR break these?

I think it would. Know that we know how to restructure the GROQ query it should be relatively easy to replace. Let me check that...

**Comment by VWSCoronaDashboard** [comment]
Should be good to go now!

**Pull request merged**